appyters/DatasetPairAttributeSimilarity/DatasetPairAttributeSimilarity.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Pair Attribute Similarity\n",
+    "## Import modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%appyter init\n",
+    "from appyter import magic\n",
+    "magic.init(lambda _=globals: _())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import gzip\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import fisher\n",
+    "from IPython.display import FileLink"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load Gene Set Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter hide_code\n",
+    "{% do SectionField(\n",
+    "    name= 'data',\n",
+    "    title= 'Upload Data'\n",
+    ")%}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Dataset 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Import Gene Set Library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%appyter code_eval\n",
+    "{% do DescriptionField(\n",
+    "    name= 'Description1',\n",
+    "    text= '<center>The examples below were sourced from <a href=\"https://maayanlab.cloud/Harmonizome/\" target =\"_blank\">Harmonizome</a>.<center>',\n",
+    "    section='data',\n",
+    ")%}\n",
+    "\n",
+    "{% set dataset1 = FileField(\n",
+    "    constraint= '.*.(txt|gmt)(.gz)?',\n",
+    "    name= 'Dataset 1',\n",
+    "    label= 'Gene Set Library 1',\n",
+    "    default= 'CCLE Cell Line Gene Expression Profiles.gmt.gz',\n",
+    "    examples= {\n",
+    "        'CCLE Cell Line Gene Expression Profiles.gmt.gz': 'https://maayanlab.cloud/static/hdfs/harmonizome/data/cclemrna/gene_set_library_up_crisp.gmt.gz'\n",
+    "    },\n",
+    "    section= 'data'\n",
+    ")%}\n",
+    "\n",
+    "gmt1 = {{dataset1}}\n",
+    "\n",
+    "if {{dataset1}}.split('.')[-1]=='gz':\n",
+    "    with gzip.open({{dataset1}}, mode='rt', encoding='utf-8', newline='\\n') as f:\n",
+    "        f = f.read().splitlines()\n",
+    "else:\n",
+    "    with open({{dataset1}}, mode='r', encoding='utf-8', newline='\\n') as f:\n",
```

Comment:
Both of these should use `gmt1` instead of re-inserting the duplicated string (dataset1)